docs/src/main/asciidoc/security-cors.adoc

```diff
@@ -11,26 +11,25 @@ include::_attributes.adoc[]
 :keywords: cors,http
 :extensions: io.quarkus:quarkus-vertx-http
 
-Cross-origin resource sharing (CORS) is an HTTP-header based mechanism that allows a server to indicate any origins (domain, scheme, or port) other than its own from which a browser should permit loading resources.
+Cross-origin resource sharing (CORS) is an HTTP header system that enables a server to specify which external origins, comprising different domains, schemes, or ports, are allowed by a web browser to access its resources.
```

Comment:
I think the original wording here was just fine, why change it?
- "Mechanism" is surely a better word than system
- the "should" word is important, it's just a recommendation for the web browser - the browser doesn't technically HAVE TO conform to it
- the word "comprising" is also a bit weird here - one origin consists of a single domain, single scheme and a single port